package-testing/python-sdk-relay/requirements.txt

```diff
@@ -1,2 +1,2 @@
 flask
-eppo-server-sdk==4.1.0
+# eppo-server-sdk installed from local wheel
```

Comment:
micro: there's an option for pip to use locally available packages, which might simplify the build file (but not sure). We use it in python ci